.github/workflows/coverage.yaml

```diff
@@ -45,6 +45,7 @@ jobs:
           coverage combine --rcfile=pyproject.toml --keep -a
           coverage report --rcfile=pyproject.toml -i --skip-empty --skip-covered --sort=cover --fail-under=90
       - name: Publish to coveralls.io
+        if: github.ref == 'refs/heads/main'
```

Comment:
Oh, also, this is random. But while looking at this branch I realized that we were published code coverage to coveralls.io on feature branches. 

That is nonsense. We want GitHub to calculate coverage during the PR, but if there are 9 commits after a PR opens, we don't need to PUBLISH the coverage 9 times.